common.mk

```diff
@@ -0,0 +1,97 @@
+# This file should be included from chart-specific Makefiles
+# Those Makefiles should define the following values BEFORE including this file:
+#
+# CHART          # Chart name (must match directory name)
+# RELEASE_NAME   # Name of helm release
+# NAMESPACE      # Optional.  Namespace used when helming.  Default is 'lightbend'
+#
+
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# NOTE:
+# Please be careful modifying this file.  Changes here will affect all of the 
+# helm-charts projects
+#
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+ifndef CHART
+    $(error CHART is not set)
+endif
+
+ifndef RELEASE_NAME
+    $(error RELEASE_NAME is not set)
+endif
+
+# Default value for namespace used with helm.  Override if required.
+NAMESPACE ?= lightbend
+
+
+define banner
+	$(info === $@)
+endef
+
+# Note: We may want to require explicit lists.  As implemented, we may end
+# up with random cruft in the packages...
+COMPONENTS := $(wildcard */.)
+SUBCOMPONENTS := $(wildcard */*/.)
+
+HELM_CHARTS_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+SCRIPTS_DIR := $(HELM_CHARTS_DIR)/scripts
+
+#####
+# Note:  There's no need to override any of these variables.
+#   (...except in the hacky enterprise-suite-latest Makefile...)
+VERSION ?= $(shell $(SCRIPTS_DIR)/export-chart-version.sh $(CHART))
+RELEASE = $(CHART)-$(VERSION)
+CHART_DIR = .
+ALL_YAML ?= all.yaml
+#####
+
+
+all: test build  ## Test then build chart
+build: init $(HELM_CHARTS_DIR)/docs/index.yaml $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML)  ## Build chart
+
+
+$(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/$(ALL_YAML): $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz
+	$(call banner)
+	mkdir -p $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)
+	helm --namespace=$(NAMESPACE) template $< > $@
```

Comment:
Would it be a good time to remove all.yaml files? They make little sense for generic usage, because we can't parameterize them.

Reply:
Fine with me.  Makes some things easier.  (I could (try to) leave them in the enterprise-suite charts for now...)